### DISCUSSION TITLE
Keyboard shortcuts not working with audio files

### DISCUSSION THREAD

**Discussion by gipadm** [smplayer-dev/smplayer#467] [General]
When playing audio files (mp3, wav, ...) the rewind, play, pause, etc. keyboard shortcuts (as defined in Preferences > Keyboard and mouse) do not work. Any fixes?

**Comment by smplayer-dev** [comment]
I cannot reproduce this problem.

**Comment by thinc22** [comment]
I was able to consistently reproduce this issue in my enviroment in the following way.
- resize the window smaller until the buttons are reduced in size, now the keyboard shortcuts don't work
- resize the window bigger until the buttons are back to normal size, now the keyboard shortcuts work again

**Comment by migdal-or** [comment]
I agree, nothing happens when I focus into Smplayer and touch SPACE or any other hotkey defined in settings if I play an audio file.
Looks like Smplayer catches the hotkeys only if there is a video part onscreen.
I can get rid of the bug for one time this way:
Options - Preferences - change multimedia engine - OK, then Smplayer restarts, shows black "video" placeholder with Smplayer logo and gets minimized to top+seek+buttons+status bar. This moment it reacts to SPACE. But if I switch to another app and return to Smplayer, it does not react to SPACE.
